eval/toxigen/run_eval.py

```diff
@@ -61,9 +62,13 @@ def main(args):
     # Generate the outputs
     if args.model_name_or_path:
         prompts = []
+        tokenizer = load_hf_tokenizer(
```

Comment:
Yeah, I see the point that a tokenizer is needed for creating the prompts now since the template function now needs the tokenizer argument.

I just feel the current logic for tokenizer/model loading and data loading/processing is not very consistent in the different evals. 

Probably it's better to separate the model loading, and tokenizer loading in the `load_hf_lm_and_tokenizer` function. For each eval, we load the tokenizer first, then the model (dependent on whether it's vllm), both of which happen before the data processing (which relies on the tokenizer).

Reply:
This is a good point, I'll make this change and re-request review.

Reply:
Seems it's done. Shall I merge it?

Reply:
Let me test it, just uploading a docker image with this code now. If it works, i'll merge?

Reply:
Sounds good! Thanks!